sharepoint/sharepoint-ps/sharepoint-online/Set-SPOApplication.md

```diff
@@ -25,13 +25,15 @@ Set-SPOApplication
 [-OwningApplicationId <OwningApplicationId>] 
 [–SharingCapability <SharingCapability>] 
 [-OverrideTenantSharingCapability <Boolean>]
+[-CopilotEmbeddedChatHosts <String>]
 ```
 
 ## DESCRIPTION
 
-`Set-SPOApplication` adjusts the sharing settings at the SharePoint Embedded application level, determining if this SharePoint Embedded application content can be shared with external guests.
+`Set-SPOApplication` cmdlet is used to set the configuration properties of a specific application, determined by the `OwningApplicationId`. Using `Sharing Capability` parameter, the administrator can adjust the sharing settings at the SharePoint Embedded application level, determining if this SharePoint Embedded application content can be shared with external guests. Using the `CopilotEmbeddedChatHosts` parameter, the host URLs that will drive the application’s chat support can be added.
```

Comment:
> Using `Sharing Capability` parameter, the administrator can adjust the sharing settings at the SharePoint Embedded application level, determining if this SharePoint Embedded application content can be shared with external guests. Using the `CopilotEmbeddedChatHosts` parameter, the host URLs that will drive the application’s chat support can be added. [](http://example.com/codeflow?start=139&length=353)

remove this extra sentence -- the points you're making here belong to the place where each parameter is documented